api/db/migrations/20220415140322_add-extension-for-pgboss.js

```diff
@@ -0,0 +1,7 @@
+exports.up = async function (knex) {
+  await knex.raw('CREATE EXTENSION IF NOT EXISTS pgcrypto;');
+};
+
+exports.down = async function (knex) {
+  await knex.raw('DROP EXTENSION IF NOT EXISTS pgcrypto;');
```

Comment:
Hello !
Il y  une petite erreur de syntaxe ici, tu n'as pas besoin du `NOT` dans  `IF NOT EXISTS`

Reply:
done